### PR TITLE
Updates Definition of a Critical User Interface in Contributing.MD

### PR DESCRIPTION
**Pull request by san7890**
## About The Pull Request

Hey there,

Right now, the current examples: "chat box", "observe button" (pretty sure), "stat panel", and "chat input" (does it mean the command line in the bottom right, or is it just the same as "chat box?) are all dated since they leverage TGUI in one way or another (or at least have HTML backups in some instances).

I decided to go back through and write a definition that might last a bit longer than anything else that we could put: administrative tooling. Let me know if the description is cringe in any way, I've just taken what I've learned from lurking and re-adapted it.

Preview how the MD would look here: https://github.com/tgstation/tgstation/blob/3d7f3f2bbdf2e7549445494bbeffff238c190bc3/.github/guides/STANDARDS.md#user-interfaces

## Why It's Good For The Game

Better definition? Something that we commonly state in Discord and the like, so may as well write it somewhere a bit more public-facing.

## Changelog

Nothing that particularly concerns players.

### CODE REVIEW

File: .github/guides/STANDARDS.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ You can avoid hacky code by using object-oriented methodologies, such as overrid
 
 ### User Interfaces
 
-* All new player-facing user interfaces must use TGUI, unless they are critical user interfaces.
-* All critical user interfaces must be usable with HTML or the interface.dmf, with tgui being *optional* for this UI.
-	* Examples of critical user interfaces are the chat box, the observe button, the stat panel, and the chat input.
+* All newly implemented player-facing user interfaces must use TGUI, unless they are critical user interfaces.
+* All critical user interfaces must be usable with HTML or the interface.dmf, with TGUI being *optional* (or even disallowed) for this UI.
+	* The most prolific example of critical user interfaces are administrative tooling UIs that must be accessible at all times during a server's life. If it's for something fluffy like the Supply Pod menu that one could "live without", it's not exactly critical. However, we really don't want something like the game panel to have a chance of not loading (The game panel is the way that one would delay/restart during a live round). When in doubt, ask.
```
Comment (LemonInTheDark) [inline .github/guides/STANDARDS.md:42]: a good definition is anything admins NEED to do their jobs, and anything someone debugging an error could NEED, including someone debugging/solving errors in tgui

Comment (LemonInTheDark) [reply .github/guides/STANDARDS.md:42]: Though I say that, nothing in the old explanation is wrong. chat and observe have backups, the stat panel just uses tgui's api logic (it's not actually React) and the chat input is bypassable with the command bar

Comment (MrStonedOne) [reply .github/guides/STANDARDS.md:42]: There should be a call out for interfaces used for recovering from error states, like the alert() box of `disable-cdn`

Comment (san7890) [reply .github/guides/STANDARDS.md:42]: They do indeed have backups- I might incorporate that in somehow.

Comment (Mothblocks) [reply .github/guides/STANDARDS.md:42]: any updates?

**Comment by san7890** [comment]
I'll do this another day

**Comment by SuperNovaa41** [comment]
Feel free to reopen this when you have time to work on it

File: .github/guides/STANDARDS.md (hunk 1 above)
Comment (Iamgoofball) [reply .github/guides/STANDARDS.md:42]: funnily enough, restarting and delay are also doable via the chatbar

**Pull request closed without merging**